cmd/benchmark/README.md

```diff
@@ -0,0 +1,71 @@
+# Benchmark
+
+The benchmark package provides a tool to benchmark Wavelet Nodes.
+
+There are two commands, `local`, and `remote`:
+- `local` command will spawn some number of Nodes locally and spam transactions on first node.
+
+- `remote` command will let you to run benchmark on an already-running Node, and spam transactions on the node.
+
+<br />
+
+You're most likely to use `remote` command as it's more flexible than `local`.
+Also, you can create your own network of Nodes to your liking and spam transactions on one of the Node in the network.
+
+## Remote
+
+Using `remote`, you can spam a batch of stake transactions, custom payload transactions, contract creation transactions,
+and poll consensus.
+
+You can combine multiple flags to spam multiple transaction types at once.
+
+### Batch of Stake Transactions
+
+The flag is `-tx`.
+
+```
+$ go run . remote -tx
```

Comment:
Care to have `-tx` flag to be true by default so that it is consistent with how benchmarks are currently ran as of now?

Reply:
@AhmadMuzakkir